### DISCUSSION TITLE
File uploads

### DISCUSSION THREAD

**Discussion by rootedsoftware** [vercel/vercel#4194] [General] [answered]
Are file uploads to API routes supported? I'm not trying to store the files on now, but I just want to get them inside my API function and then upload them to a 3rd party.

**Comment by paulogdm** [comment, marked as answer]
Hello,

Yes, it is supported without any issues. It is just not very efficient since you could stream the file directly to most providers with signed requests.

**Reply by IRediTOTO** [reply]
But that will make run API on client, that's look not well, right ?

**Reply by IRediTOTO** [reply]
btw do you know what is the limit of file upload ?
